packages/traceability-schemas/schemas/CargoItem.json

```diff
@@ -0,0 +1,86 @@
+{
+  "$id": "https://w3id.org/traceability/schemas/CargoItem.json",
+  "$schema": "https://json-schema.org/draft-07/schema#",
+  "$comment": "{\"term\": \"CargoItem\", \"@id\": \"https://service.unece.org/trade/uncefact/vocabulary/uncefact/#ConsignmentItem\"}",
+  "title": "Cargo Item",
+  "description": "Addresses the cargo items to be stuffed into a container for a shipment. A cargo item cannot be split across containers.",
```

Comment:
```suggestion
  "description": "Addresses the cargo items to be loaded into a container for a shipment. A cargo item cannot be split across containers.",
```

Reply:
Withdrawn, because "stuffed" is apparently industry standard language...

I can't "resolve" this thread; anyone else who has the relevant button may do so.